corehq/apps/export/templates/export/partials/new_customize_export_templates.html

```diff
@@ -298,7 +298,9 @@ <h4>{{ terminology.choose_fields_label }}</h4>
                                  as: 'tag'
                                }">
                 <span data-bind="text: tag,
-                                 attr: { class: $root.getTagCSSClass(tag) }"></span>
+                                 attr: { class: $root.getTagCSSClass(tag) },
+                                 visible: column.formatProperty() !== 'name'">
```

Comment:
Ideally we'd do this a little deeper in the code, but after looking around for 10 minutes, the effort there doesn't seem worth it, so I'm okay with this last-minute approach.

I do think we should make this condition more airtight though. (1) we only want to exclude the deleted tag, and (2) we only want to exclude the `name` case property, not any ExportColumn who's display string is "name".

You can make the `TAG_DELETED` constant available as `$root.constants.TAG_DELETED` by adding `self.constants = constants` right below this line: https://github.com/dimagi/commcare-hq/blob/a9f473c51a9c5e34afad2171f16dcb176b29f21f/corehq/apps/export/static/export/js/models.js#L95.

And you can check whether a column corresponds to the case name with `column.item.isCaseName()`.

I think both those suggestions should work, but you may have to play around with it a bit. I'm then thinking you could exclude the tag if `tag == $root.constants.TAG_DELETED && column.items.isCaseName()` or something like that.

Finally, this is a minor thing, but instead of just making it invisible, you can make the DOM element not appear at all by using the
``
<!-- ko: if (<condition>) -->
...
<!-- /ko -->
```
syntax. (You can see other examples in the same file.)

Reply:
Thanks Danny, for the review.
I will try it in the way you suggested. 

I took `code never show the “deleted” tag next to the “name” property in case exports` this statement too literally in the code. 😅